### PR TITLE
Add "New in Talk" entries for version 12

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/nextcloud/spreed/issues/6043

### CODE REVIEW

File: lib/Chat/Changelog/Manager.php [hunk 1]
```diff
@@ -115,6 +115,10 @@ public function getChangelogs(): array {
 			$this->l->t('- Give your conversations some context with a description and open it up so logged in users can find it and join themselves'),
 			$this->l->t('- See a read status and send failed messages again'),
 			$this->l->t('- Raise your hand in a call with the R key'),
+			$this->l->t('New in Talk %s', ['12']),
+			$this->l->t('- Join the same conversation and calls from multiple devices'),
```
Comment (nickvergessen) [inline lib/Chat/Changelog/Manager.php:119]:
```suggestion
			$this->l->t('- Join the same conversation and call from multiple devices'),
```
or
```suggestion
			$this->l->t('- Join the same conversations and calls from multiple devices'),
```

File: lib/Chat/Changelog/Manager.php [hunk 2]
```diff
@@ -115,6 +115,10 @@ public function getChangelogs(): array {
...
+			$this->l->t('- Send voice messages, share your location or contact details'),
+			$this->l->t('- Add groups to a conversation and new group members will automatically become participants'),
```
Comment (nickvergessen) [inline lib/Chat/Changelog/Manager.php:121]:
```suggestion
			$this->l->t('- Add groups to a conversation and new group members will automatically be added as participants'),
```

**Comment by PVince81** [comment]
@nickvergessen adjusted

**Review by marcoambrosini — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**